RELEASE_NOTES.md

```diff
@@ -42,6 +42,8 @@ when updating from the currently stable 2.4.x series, please bear in mind that y
 - Add new raster mask supports. This is a copy of a parametric mask
   which is stable during the whole pixel-pipe.
 
+- Modules can now be re-ordered with shift+drag.
```

Comment:
It is not good to add RELASE_NOTES unrelated to current PR because if there is troubles and we revert it will also revert an needed information. And if we need to merge in a branch a feature and not the other this will require editing the commits before merging.

So, please in the future keep this in mind. Thanks.

Reply:
I didn't thought about that, I'll move this to another PR.